ARAnalytics.podspec

```diff
@@ -5,7 +5,7 @@ Pod::Spec.new do |s|
   s.homepage     =  'https://github.com/orta/ARAnalytics'
   s.authors      =  { 'orta' => 'orta.therox@gmail.com', 'Daniel Haight' => "confidence.designed@gmail.com" }
   s.source       =  { :git => 'https://github.com/orta/ARAnalytics.git', :tag => s.version.to_s }
-  s.ios.deployment_target = "7.0"
+  s.ios.deployment_target = "8.0"
```

Comment:
This is a semver breaking change, we'll need to bump the major for the next release